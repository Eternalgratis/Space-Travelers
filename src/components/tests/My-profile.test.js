import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/configureStore';
import MyProfile from '../My-Profile';

describe('to test the rockets elements', () => {
  it('matches DOM rockets snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MyProfile />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
