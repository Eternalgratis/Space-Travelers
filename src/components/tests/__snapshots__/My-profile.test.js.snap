// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to test the rockets elements matches DOM rockets snapshot 1`] = `
<div
  className="myprofileContainer"
>
  <div
    className="rocketDiv"
  >
    <h2>
      My Rockets
    </h2>
    <p
      className="rocketDesc"
    >
      No Rockets
    </p>
  </div>
  <div
    className="dragonDiv"
  >
    <h2>
      My Dragons
    </h2>
    <p
      className="dragonDesc"
    >
      No Dragons
    </p>
  </div>
  <div
    className="missionsDiv"
  >
    <div>
      <h2>
        My Missions
      </h2>
      <div
        className="content"
      >
        <p
          className="missionDesc"
        >
          No Missions
        </p>
      </div>
    </div>
  </div>
</div>
`;
